android/build.gradle

```diff
@@ -47,6 +47,10 @@ android {
     sourceCompatibility JavaVersion.VERSION_1_8
     targetCompatibility JavaVersion.VERSION_1_8
   }
+
+  kotlinOptions {
```

Comment:
Could you explain, why jvm target has been changed ?

Reply:
When I implemented  "androidx.core:core-ktx:1.6.0", it had a build error. So, I searched and found a solution in StackOverflow. I can't remember exactly what it is. Sorry.